packages/backend/src/app/helper/engine-helper.ts

```diff
@@ -18,13 +19,27 @@ export const engineHelper = {
             workerToken: await workerToken({ collectionId: operation.collectionId, projectId: operation.projectId })
         }) as ExecutionOutput;
     },
-    async executeTrigger(operation: ExecuteTriggerOperation): Promise<void | unknown[]> {
+    async executeParseEvent(operation: ExecuteParseEventOperation): Promise<ParseEventResponse> {
+        const sandbox = sandboxManager.obtainSandbox();
+        let result;
+        try {
+            await sandbox.cleanAndInit();
+            result = await execute(EngineOperationType.EXTRACT_EVENT_DATA, sandbox, operation);
```

Comment:
This will need to be updated to install the piece dependency after merging the pieces versioning PR